.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - '3.8'
 # command to install dependencies
 install:
+  - pip install .
```

Comment:
I don't think this is needed. Tox will install the package itself.

Reply:
sure! Got rid of it.